elftools/elf/elffile.py

```diff
@@ -46,6 +47,12 @@ class ELFFile(object):
     """ Creation: the constructor accepts a stream (file-like object) with the
         contents of an ELF file.
 
+        Optionally, a stream_loader function can be passed as the second
```

Comment:
Be a bit more prescriptive in describing the function: it takes a XXX and returns a XXX 

Also nit: s/pathes/paths/